projects/adrv9001/builds.json

```diff
@@ -0,0 +1,40 @@
+{
+      "xilinx": {
+            "demo": {
+                  "flags": "",
+                  "hardware": [
+                        "adrv9001_zc706",
+                        "adrv9001_zcu102",
+                        "adrv9001_zcu102_lvds",
+                        "adrv9001_zed"
```

Comment:
We do have HDL that has support for ZED + ADRV9001 but the no-OS project was never updated to support that.
zc706 is also with question mark.

Reply:
Will be update or should be removed from the build?

Reply:
removed zed and zc706 for the moment